pibooth/booth.py

```diff
@@ -423,7 +427,8 @@ def main():
         LOGGER.info("Listing all fonts available...")
         print_columns_words(get_available_fonts(), 3)
     elif options.reset:
-        config.save()
+        config.save(True)
```

Comment:
Same here to be easier to read I would put
> config.save(default=True)
even if I agree it is the same

Reply:
done